src/components/Input/Input.tsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import "./Input.css";
+
+type InputProps = {
+  value: string;
+  placeholder: string;
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void;

```

Comment:
лучше сразу сделать так, чтобы `onChange` принимал 
```suggestion
  onChange: (value: string) => void;
```
Для этого надо внутри инпута создать функцию `const handleChange = (e: React.ChangeEvent<HTMLInputElement>) => onChange(e.target.value)`
Нужно это все для того, чтобы не писать каждый раз `e.target.value` при использовании данного компонента